### ISSUE TITLE
Internationalize feedback button

### ISSUE THREAD

**Issue by anxolin** [gnosis/gp-swap-ui#640]
Appzi allows to internationalize the button and feedback.

Add literals.

**Comment by annamsgeorge** [comment]
What would change for the user? Some users are already texting us in another language and we can translate via google translate :D And I think the buttons are quite clear, no? It has a happy smiley, a sad smiley and a bulb for ideas :D

**Comment by anxolin** [comment]
Yes, is not important. For context, this comes from a meeting from Appzi CEO about some security concerns we had. He addressed them, and mentioned that if we provide literals they can also translate. 

I would only do this one if we translate to other languages, I added it here just have it in the backlog.
